contrib/babelfishpg_tsql/src/backend_parser/kwlist.h

```diff
@@ -383,6 +383,7 @@ PG_KEYWORD("readuncommitted", TSQL_READUNCOMMITTED, UNRESERVED_KEYWORD)
 PG_KEYWORD("real", REAL, COL_NAME_KEYWORD)
 PG_KEYWORD("reassign", REASSIGN, UNRESERVED_KEYWORD)
 PG_KEYWORD("recheck", RECHECK, UNRESERVED_KEYWORD)
+PG_KEYWORD("recompile", TSQL_RECOMPILE, RESERVED_KEYWORD)
```

Comment:
we marked TSQL_RECOMPILE token as unreserved keyword in tsql-rule but we are marking it to reserved keyword. Either of two is placed incorrectly. Please fix that and we also need to add test for this.

Reply:
Changed to unreserved. There is already a test case for this: BABEL-2730.sql